### PR TITLE
Cherry Arcade Migration for release/3.1

### PR DESCRIPTION
**Pull request by deepchoudhery**
Migration for arcade build.

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-        "version": "2.2.102"
-    },
-    "msbuild-sdks": {
-        "Internal.AspNetCore.Sdk": "2.2.1-build-20190219.3"
-    }
+  "tools": {
+    "dotnet": "3.0.100"
+  },
+  "msbuild-sdks": {
+    "Microsoft.DotNet.Arcade.Sdk": "5.0.0-beta.19480.3"
```
Comment (vijayrkn) [inline global.json]: These versions should be 3.1 specific.

File: eng/Versions.props [hunk 2]
```diff
@@ -69,7 +92,7 @@
     <MicrosoftAspNetCoreIdentityUIPackageVersion>3.0.0</MicrosoftAspNetCoreIdentityUIPackageVersion>
 
     <!-- Microsoft.Extensions.Identity.Stores -->
-    <MicrosoftExtensionsIdentityStoresPackageVersion>3.1.0-preview1.19508.20</MicrosoftExtensionsIdentityStoresPackageVersion>
```
Comment (vijayrkn) [inline eng/Versions.props]: All the versions should be same as before. i.e 3.1 dependencies.

File: global.json (hunk 1 above)
Comment (deepchoudhery) [reply global.json]: 3.1 preview 1 or 2?

Comment (deepchoudhery) [reply global.json]: using preview2 for now

**Pull request merged**